packages/plugins/i18n/admin/src/components/CMHeaderActions.tsx

```diff
@@ -278,6 +279,10 @@ const FillFromAnotherLocaleAction = ({
     onClose();
   };
 
+  if (!hasI18n || !Array.isArray(locales) || locales.length === 0) {
```

Comment:
if i18n is enable but the locales aren't loaded, should we fall back to the disabled state defined below? otherwise I don't see what it's for

Reply:
good point, will do that!